### DISCUSSION TITLE
[FAIL. Reason: setup failed: a broadcast is active already] setUp() (gas: 0)

### DISCUSSION THREAD

**Discussion by king-null** [Cyfrin/foundry-full-course-cu#1517] [Q&A] [answered]
i try to run a forge test in lesson 7 and i got thiis error saying a broadcast is active already, what could be the problem

**Comment by EngrPips** [comment, marked as answer]
Hello @king-null , You would normally get this error whenever you use a `vm.startBroadcast()` and did not end the broadcast with `vm.stopBroadcast()` yet and then attempt to start another broadcast. Basically when you have 2 `vm.startBroadcast()` without a `vm.stopBroadcast()` between them  in a process of transaction.

Please try to look  at your script and test closely and be sure there is no such point where that happens. If you can't find it  maybe push the code to github and share me a link to it so i set up your repo locally on my machine and try help you find it. They can be difficult to catch sometimes

**Reply by king-null** [reply]
thanks so much @EngrPips, i appreciate your help

**Reply by EngrPips** [reply]
Anytime friend, i am glad i could help
